src/components/dashboardV2/MembersList.vue

```diff
@@ -14,6 +14,7 @@
           <b v-if="member.bar_id === activeBarId">{{ member.first_name }} {{ member.last_name }}</b>
```

Comment:
A couple of things:

1.- Read the comment at line 10.
If you want to render the CM as a member, you need to update the if clause to be ">0" or (my preferred) to get rid of the complete if statement at line 11 (and the related else block at line 32/33) so we don't check for the amount of people in the community.

2.- Beyond that, I see an error in the console while trying to render the CM in the list of members. And this is the information that the server returns about the CM: 
`
{"id":"910c177d-fc8a-4ded-99b0-c6cf45c0bf41","first_name":"Pat","last_name":Smith,"role":"manager","bar_id":null,"state":"active"}
`
As you can see, the CM doesn't have a bar assigned, and I assume that we need to add the default one to get rid of such errors. I'd say we need to update the CM's profile as soon as the default bar is created.

Reply:
Good catch! I've fixed as per your recommendation on #1. For #2 I've done a duplicate fix of in the dashboard... It probably should have been resolved somewhere earlier and members passed around from a central point, but with the timecrunch I think this is a sensible solution (it's only a 1-liner that is duplicated)

Reply:
ok, did a further update... It looks like members are fetched in two places, fixed the bar_id thing where data is loaded in both places.. I dont think this solution is too crazy - and a more detailed fix would be too extensive with the given timeframe